app/renderer/assets/locales/en-US/contract.json

```diff
@@ -31,5 +31,6 @@
   "interface_tables_placeholder": "Select a table...",
   "interface_tables_scope": "Table Scope (Account Name)",
   "interface_tables_subheader": "Smact Contracts on the EOS network can contain tables containing public information.",
-  "tables": "Tables"
+  "tables": "Tables",
+  "wallet_panel_wallet_signbroadcast": "Broadcast transaction"
```

Comment:
??

Reply:
LOL kind of unrelated to this PR, but that key was being called on the contract interface and missing on this file, therefore I added it.

Reply:
That's already an existing key in the `wallet.json` file. Instead of duplicating the localization string (which means it would have to be translated again), you can instead just reference a different file from the translation string.

When using: 

```
{t('wallet_panel_wallet_signbroadcast')}
```

It will load the `wallet_panel_wallet_signbroadcast` string out of the default file specified for the component, but if you do:

```
{t('wallet:wallet_panel_wallet_signbroadcast')}
``` 

It forces it out of the default context and uses the `wallet.json` file instead of whatever the default file was for that string. This works much better and doesn't duplicate strings :)

Reply:
Awesome! That will come in handy!